kpops/cli/main.py

```diff
@@ -220,8 +220,8 @@ def generate(
         for component in steps_to_apply:
             component.template(api_version, ca_file, cert_file)
     elif cert_file or ca_file or api_version or steps:
-        raise TypeError(
-            "The following flags can only be used in conjuction with `--template`: \n \
+        log.warning(
```

Comment:
Can the warning also cause issues? If yes, I will just remove the else entirely.

Reply:
No the warning does not cause an issue :)